docs/byte-array-converter.adoc

```diff
@@ -0,0 +1,34 @@
+:page-title: Convert Number Argument to Byte Array
+:page-description: The JUnit 5 (Jupiter) extension `@NumberToByteArrayConversion` convert the number value of an argument to its byte array representation
```

Comment:
What about `argument converter` instead of `extension`?

My concern is the terminology clash with [`Extension`](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/extension/Extension.html).

Applies also to other sentences in this file.

Reply:
Superseded by #751.